CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The change log describes what is "Added", "Removed", "Changed" or "Fixed" betwee
 - Autowiring support for FlexibleClient, HttpMethodsClientInterface and
   BatchClientInterface if they are enabled on the default/first client.
   (Only available with Httplug 2)
+- Add configuration option to disable registration of default client
```

Comment:
i find it useful when the changelog is specific so i can find the thing it talks about. can we make this "Configuration option `register_default_client` to allow disabling registering a default client"

Reply:
this should also mention autowiring somehow

Reply:
Done.